frontend/pages/labels/ManageLabelsPage/LabelsTable/LabelsTableConfig.tsx

```diff
@@ -131,9 +129,10 @@ const generateTableHeaders = (
         />
       ),
       accessor: "label_membership_type",
-      Cell: (cellProps: ICellProps) => (
-        <TextCell value={capitalize(cellProps.cell.value)} />
-      ),
+      Cell: (cellProps: ICellProps) => {
+        const type = cellProps.row.original.label_membership_type;
+        return <TextCell value={LabelMembershipTypeToDisplayCopy[type]} />;
```

Comment:
In the event the backend adds a new membership type, does this end up displaying "undefined"? As a form of "failing loudly" that's probably ok.

Reply:
Good point. I'd assume whatever feature would introduce such a new type would have a corresponding frontend component part of which would involve the engineer updating this mapping and the related index type, but if for whatever reason that didn't line up, undefined seems pretty loud, yes.

Fantasizing about a single config json or something that gets translated to corresponding backend structs and frontend types ☁️